src/main/java/ch/wisv/events/utils/ICalendarBuilder.java

```diff
@@ -0,0 +1,79 @@
+package ch.wisv.events.utils;
+
+import biweekly.ICalendar;
+import biweekly.component.VEvent;
+import biweekly.property.Summary;
+import ch.wisv.events.core.model.event.Event;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Copyright (c) 2016  W.I.S.V. 'Christiaan Huygens'
+ * <p>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * <p>
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+public class ICalendarBuilder {
+
+    /**
+     * Generates a iCal object for a list of events
+     * @param events
+     * @return
+     */
+    public static ICalendar createIcalEventList(List<Event> events){
+        ICalendar iCal = new ICalendar();
+        events.forEach(event -> iCal.addEvent(createICalEvent(event)));
+
+        iCal.addName("CH Event Calendar");
+        iCal.setColor("turquoise");
```

Comment:
Let's not start with all the self hypes and just use the one of the default CH colours.